guides/getting-started/getting-started-operator-kubernetes.adoc

```diff
@@ -1,5 +1,9 @@
 :guide-title: Kubernetes Operator
-:guide-summary: Get started with Keycloak Operator on Kubernetes
+:guide-summary: Get started with Keycloak Operator on Kubernetes [Wildfly distribution]
+
+.Note:
+[NOTE]
+This guide refers to the Wildfly distribution of Keycloak.
```

Comment:
I think we should be even more explicit. Something like:
```suggestion
This guide refers to the https://github.com/keycloak/keycloak-operator[old operator] and Wildfly distribution of Keycloak.
```
WDYT?

Reply:
i'd say yes, and also perhaps add "legacy" somewhere, and point people to the latest greatest one? e.g. 
```
This guide refers to the https://github.com/keycloak/keycloak-operator[old, unmaintained operator] and legacy Wildfly distribution of Keycloak. The guide for the new operator can be found https://...[here].

```
?

Reply:
I would not use "unmaintained" wording. :) It is maintained, just deprecated.

Reply:
right :) deprecated is better. that was just what came to my mind